tests/components/google/conftest.py

```diff
@@ -132,9 +132,16 @@ async def token_scopes() -> list[str]:
 
 
 @pytest.fixture
-async def creds(token_scopes: list[str]) -> OAuth2Credentials:
+async def token_expiry() -> datetime.datetime:
```

Comment:
Many of these fixtures don't need to be run in the event loop, I think.
```suggestion
def token_expiry() -> datetime.datetime:
```

Reply:
(The lines I am touching in this PR I am cleaning up)